CONTRIBUTING.md

```diff
@@ -104,41 +103,48 @@ variables:
 
```

Comment:
Might be worth mentioning here that after running the script, environment vars won't be picked up by visual studio until it's restarted.  This may even apply to powershell or command prompt that you are using if running the build script too?